reddit2telegram/cron_app.py

```diff
@@ -31,22 +53,12 @@ def read_own_cron(own_cron_filename, config):
             diff_seconds = diff.total_seconds()
             if 0.0 <= diff_seconds and diff_seconds <= 59.9:
                 list_of_processes_to_start.append(row['submodule_name'])
-        cycles = 0
-        random.shuffle(list_of_processes_to_start)
-        for process_to_start in list_of_processes_to_start:
-            successfully_started = False
-            while not successfully_started:
-                cycles += 1
-                cycles_factor = math.ceil(cycles / 45)
-                time.sleep(1)
-                free_memory_mb = psutil.virtual_memory().free / 1024**2
-                if free_memory_mb > free_memory_constant * min(cycles_factor, 3):
-                    supplying_process = Process(target=supply, args=(process_to_start, config))
-                    supplying_process.start()
-                    successfully_started = True
-                    break
-                else:
-                    successfully_started = False
+    random.shuffle(list_of_processes_to_start)
+    executor = _create_thread_pool(config)
+    executor.map(
+        lambda submodule_name: _process_single_entry(submodule_name, config),
```

Comment:
Should here be 3 args?

Reply:
Or argument by default as suggested above.

Reply:
You are right. Changed to the default argument.